index.js

```diff
@@ -1,24 +1,65 @@
 'use strict';
 const pathExists = require('path-exists');
 const modifyFilename = require('modify-filename');
+const slash = require('slash');
```

Comment:
I'm not sure if `slash` should be used 🤔 Maybe if the user wants unixy paths he should normalize them himself? I think this would align more with the built-in `path` module.

Reply:
`slash` is removed